### DISCUSSION TITLE
Use Annotations with CDK

### DISCUSSION THREAD

**Discussion by John0x** [aws/aws-lambda-dotnet#1230] [Q&A] [answered]
Hey, is it possible to deploy a project that is using the new annotations using CDK?

Something like the "Cloudformation Include" thing but for SAM

**Comment by ashishdhingra** [comment]
@John0x Could you please share the sample code solution for your scenario? Are you trying to deploy .NET Lambda project using CDK tooling? If yes, then probably move this discussion to https://github.com/aws/aws-cdk.

**Comment by normj** [comment, marked as answer]
@John0x We don't currently have a smooth integration with CDK but it something we want to add to the annotations library.
